packages/blockchain-sdk-aptos/src/encryption/encryption-keys-provider.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  DiffeHellmanKeys,
+  EncryptionKeysProvider,
+  UnsupportedOperationError,
+} from '@dialectlabs/sdk';
+import type { DialectAptosWalletAdapterWrapper } from '../wallet-adapter/dialect-aptos-wallet-adapter-wrapper';
+
+export class DialectAptosWalletAdapterEncryptionKeysProvider extends EncryptionKeysProvider {
+  constructor(
+    private readonly dialectWalletAdapter: DialectAptosWalletAdapterWrapper,
+  ) {
+    super();
+  }
+
+  async getFailSafe(): Promise<DiffeHellmanKeys | null> {
+    return null;
```

Comment:
Is this supposed to be a blank function?

Reply:
yes, encryption not supported in aptos

Reply:
Got it.